src/app/lib/config/services/uzbek.ts

```diff
@@ -32,337 +32,650 @@ const baseServiceConfig = {
+            reject: 'Йўқ, мени ўзгартириш саҳифасига етакланг',
+            rejectUrl:
+              'https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
           },
         },
-        canonical: {
-          title: 'Кукис(Cookies)га рози эканингизни билдиринг',
+      },
+      media: {
+        noJs: 'Мултимедиа ўйнаш бу қурилмада дастакланмайди',
+        contentExpired: 'Бу контентни ортиқ тинглаб/томоша қилиб бўлмайди.',
+        contentNotYetAvailable:
+          'Бу контент ҳали тинглаш/томоша қилишга тайёр эмас.',
+        audio: 'Аудио',
+        photogallery: 'Фото галерея',
+        video: 'Видео',
+        bbc_uzbek_radio: {
+          title: 'BBC Uzbek Radio Dasturi',
+          subtitle: 'O’zbekiston, mintaqa va dunyo yangiliklari O’zbek tilida',
+        },
+        bbc_uzbek_tv: {
+          title: 'Bi-bi-si O‘zbek - Afg‘oniston uchun TV dastur',
+          subtitle:
+            'Dastur haftada 5 kun - dushanbadan juma kuniga qadar Toshkent vaqti bilan soat 18.30 -18.40 da efirga uzatiladi',
+        },
+        listen: 'Тингланг',
+        watch: 'Кўринг',
+        listenLive: 'Жонли эфирда тингланг',
+        liveLabel: 'ЖОНЛИ ЭФИР',
+        nextLabel: 'NEXT',
+        previousRadioShow: 'Аввалги радио дастур',
+        nextRadioShow: 'Кейинги радио дастур',
+        duration: 'Давомийлиги',
+        recentEpisodes: 'Олдинги дастурлар',
+      },
+      socialEmbed: {
+        caption: {
+          textPrefixVisuallyHidden: 'Видео тагсўзи, ',
+          text: 'Огоҳлантириш:Учинчи манба материалида реклама бўлиши мумкин',
+          articleText:
+            'Огоҳлантириш: Би-би-си ташқи саҳифалардаги контент учун масъул эмас',
+          articleAdditionalText:
+            '%provider_name% бу контентда реклама бўлиши мумкин',
+        },
+        fallback: {
+          text: 'Бу материалга кириш имконсиз',
+          linkText: 'Кўпроқ кўринг %provider_name%',
+          linkTextSuffixVisuallyHidden: ', ташқи',
+          warningText: 'Би-би-си ташқи сайтлар мазмуни учун масъул эмас.',
+        },
+        skipLink: {
+          text: 'Ўтказиб юборинг %provider_name% пост ',
+          endTextVisuallyHidden: 'Охири %provider_name% пост',
+        },
+        consentBanner: {
+          heading: `[social_media_site] контентига рухсат бериш`,
+          body: `Айни мақолада [social_media_site] томонидан тақдим қилинган контент мавжуд. Биз бу контент юкланмасидан аввал сизнинг розилигингизни сўраймиз, чунки улар куки ва бошқа технологиялардан фойдаланган бўлиши мумкин. Сиз   [social_media_site] [link] ҳаволасида кукиларга доир  [/link] ва шахсий маълумотларга  оид қоидалар ҳақида аввал ўқиб,  кейин қабул қилишга рози бўлишингиз  мумкин. Кўриш учун “қабул қилиш ва давом этиш”ни танланг.`,
+          button: 'Қабул қилинг ва давом этинг',
+        },
+      },
+      include: {
+        errorMessage:
+          'Sorry, we can’t display this part of the story on this lightweight mobile page.',
```

Comment:
can we get a translation from editorial on this before merging? Not urgent

Reply:
About half of the services have this in English. I can ask editorial if they have the translation to update it though.